glossary/public-code-definition.md

```diff
@@ -8,6 +8,6 @@ Public code is both civic code (like policy or regulation) and computer source c
 
 Because public code serves the public interest, it should be: open, legible, accountable, accessible and sustainable.
 
-By developing public code independent from but implementable in a local context, as well as documenting the development process openly, it can provide a building block for others to re-implement in their local context, take as a starting point to continue development or as a basis for learning.
+Public code can be building blocks for others to re-use and re-implement in their local context, creating their public code. It can serve as a starting point to continue development or as a basis for learning. To enable that, the development of public code needs to be independent from, but implementable in, those contexts. The development process must also be openly documented.
```

Comment:
```suggestion
Public code can provide a building block for others to:

* use as a basis for learning
* take as a starting point to continue development
* re-implement in their local context

To enable this, public code development needs to be independent from, but implementable in all of the contexts listed above. The development process must also be openly documented.
```

Reply:
This sounds confusing to me, as it seems like the coe has agency providing something else rather than being the building block itself (which I still believe is the point we want to make, right?).

Reply:
I've updated this - is it clearer now? I'm trying to avoid @lscheske's 'comprise' suggestion because (for me) that suggests that the building blocks are part of a larger set that we're not explaining.